setup.py

```diff
@@ -53,6 +53,9 @@
     packages=find_packages("src", exclude=["tests"]),
     package_dir={"": "src"},
     python_requires=">=3.8",
+    install_requires=[
+        "legacy-cgi~=2.6; python_version >= '3.13'",
```

Comment:
I feel like `~=2.6` is too tight, would prefer to only pin a lower bound.

Reply:
Alright, done.